api/src/main/java/org/pmiops/workbench/config/WebMvcConfig.java

```diff
@@ -90,4 +82,14 @@ static ServletContext getRequestServletContext() {
         .getRequest()
         .getServletContext();
   }
+
+  @Override
+  public void addCorsMappings(CorsRegistry registry) {
+    registry
+        .addMapping("/**")
+        .allowedMethods("GET", "POST", "PUT", "DELETE", "OPTIONS")
```

Comment:
CorsInterceptor: `response.setHeader(METHODS_NAME, "GET, HEAD, POST, PUT, DELETE, PATCH, TRACE, OPTIONS");`

Looks like we're missing HEAD, PATCH, and TRACE here.